PokerGame/PokerEngine/Poker.cs

```diff
@@ -80,6 +133,23 @@ private static bool HasPair(List<Card> hand, out Rank pair)
             return false;
         }
 
+        private static bool HasTwoPair(List<Card> hand, out Rank pairOne, out Rank pairTwo)
+        {
+            var handICanManipulate = new List<Card>(hand); 
```

Comment:
rename: cardsWithoutFirstPair

Reply:
awesome was meant to rename but forgot 👍